eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -0,0 +1,11 @@
+<!-- When altering this file or making other Source Build related changes, include @dotnet/source-build as a reviewer. -->
+<!-- See aka.ms/dotnet/prebuilts for guidance on what pre-builts are and how to eliminate them. -->
+
+<UsageData>
+  <IgnorePatterns>
+     <!-- These are coming in via runtime but the source-build infra isn't able to automatically pick up the right intermediate. -->
+    <UsagePattern IdentityGlob="Microsoft.NET.ILLink.Tasks/*9.0.*" />
+    <!-- This is a source only nuget package, there are no dlls coming to the build from this. -->
+    <UsagePattern IdentityGlob="Polyfill/*" />
```

Comment:
@MichaelSimons this package only contains source code. There are no pre-builds in it.

Reply:
This is actually great.  Since this is a source code only package then we should add it to SBRP as a text-only package.  This is a lot easier than adding it as a source build external. See https://github.com/dotnet/source-build-reference-packages/blob/main/README.md#text-only

Reply:
https://github.com/dotnet/source-build-reference-packages/pull/1095